src/main/java/core/basesyntax/dao/impl/MessageDaoImpl.java

```diff
@@ -12,21 +14,62 @@ public MessageDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Message create(Message entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
```

Comment:
let's keep our code consistent, use either `save()` or `persist()` everywhere and corresponding cascade type where needed

Reply:
Can't i use both ?

Reply:
It does not work, when i use save

Reply:
https://www.baeldung.com/jpa-cascade-types#2-hibernate-cascade-type